Container Security/kernel-eks-implementation-guide.md

```diff
@@ -172,15 +208,37 @@ You will need AWS credentials and docker tool installed locally.
  - (optional) Verify that falcon-node-sensor pod running. You should see one pod per each node on your cluster.
    ```
    $ kubectl get pods -n falcon-system
+   ```
+   Example output:
+   ```
    NAME                              READY   STATUS    RESTARTS   AGE
    falcon-helm-falcon-sensor-bs98m   2/2     Running   0          21s
    ```
  - (optional) Verify that given pod has registered with CrowdStrike Falcon and received unique identifier.
    ```
-   $ kubectl exec -n falcon-system falcon-helm-falcon-sensor-bs98m -c falcon-node-sensor -- falconctl -g --aid
-   aid="a582XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+   $ IFS=$'\n'
+     for i in $(kubectl get pods -n falcon-system | awk 'FNR > 1' | awk '{print $1}')
+     do echo "$i - $(kubectl exec $i -n falcon-system -c falcon-node-sensor -- falconctl -g --aid)"
+     done
+   ```
+   Example output:
+   ```
+     falcon-helm-falcon-sensor-XXXX - aid="a582XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+   ```
+ - (optional) Check the Reduced Functionality Mode state of the Falcon Sensor.
+ - Note that the value returned should be false if running on supported kernel and platform versions.
+   ```
+   $ IFS=$'\n'
```

Comment:
If we can rework the commands so that we don't have to handle IFS, that would be great.

Reply:
I think I reworked the awk earlier in this command after the IFS. It seems to work without it now as the single pod is returned to the next command and not the line.